addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -366,9 +379,32 @@ function factory(dependencies) {
             default: null,
         }),
         renamingThreads: one2many('mail.thread'),
-        replyingToMessage: one2one('mail.message', {
+        /**
+         * Useful for the reply feature in Inbox. There is only one reply
+         * composer shown at a time, which depends on the message set to be
+         * replied to thanks to this field.
```

Comment:
Missing a 1st sentence to explain the "what" (other newly introduced fields have this 1st sentence, you can take inspiration from them)

Reply:
Ok I changed, the information was there but not in a sentence written as we would expect.